services/src/main/java/org/keycloak/authentication/authenticators/x509/X509AuthenticatorConfigModel.java

```diff
@@ -253,6 +253,15 @@ public X509AuthenticatorConfigModel setCanonicalDnEnabled(boolean value) {
         getConfig().put(CANONICAL_DN, Boolean.toString(value));
         return this;
     }
+
+    public boolean isCertValidationEnabled() {
+        return Boolean.parseBoolean(getConfig().get(TIMESTAMP_VALIDATION));
```

Comment:
For existing users that upgrade to a new release TIMESTAMP_VALIDATION is not set, so this should make sure it works when getConfig().get(TIMESTAMP_VALIDATION) is null

Reply:
done